src/phoenix/db/models.py

```diff
@@ -66,7 +66,7 @@ class Base(DeclarativeBase):
         naming_convention={
             "ix": "ix_%(column_0_label)s",
             "uq": "uq_%(table_name)s_%(column_0_name)s",
-            "ck": "ck_%(table_name)s_`%(constraint_name)s`",
+            "ck": "ck_%(table_name)s_%(constraint_name)s",
```

Comment:
The SQLAlchemy docs have backticks, presumably in case the `constrain_name` includes spaces. But this results in the following:

```
CONSTRAINT "ck_span_annotations_`valid_annotator_kind`" CHECK (annotator_kind IN ('LLM', 'HUMAN'))
```

Reply:
Yeah - is that a problem?

Reply:
Not as far as I can tell. Just extra characters. Keep that way?

Reply:
added back in e66dd62f1e0eecefe896376099677bb608923496